src/main/resources/db/changelog/003_Add_table_applicants.sql

```diff
@@ -0,0 +1,12 @@
+-- liquibase formatted sql
+-- changeset author:oskachkov
+
+CREATE TABLE applicants(
+    user_id INTEGER NOT NULL ,
+    first_name VARCHAR(255) NOT NULL ,
+    last_name VARCHAR(255) NOT NULL ,
+    social_security_number INTEGER NOT NULL ,
```

Comment:
As I know social_security_number in fact is a varchar

Reply:
it can be different in different counties. We don't have any description for this field, so we can use any type of field by default.